travvy/src/main.js

```diff
@@ -3,6 +3,15 @@ import App from './App.vue'
 import router from './router'
 import {sync} from 'vuex-router-sync'
 import store from './store/store.js'
+import * as VueGoogleMaps from 'vue2-google-maps'
+
+Vue.use(VueGoogleMaps, {
+  load:{
+    key: "AIzaSyBre0sAE_zK3rJcFYStwk9QhfrsJAKazUI"
```

Comment:
Does this key last forever? It'd be interesting to know how long this key for the api is good for.

Reply:
This key is linked to my google account. I have a one year free trial with $300 of credit. If we reach the limit we have to start paying.